frontend/src/views/login/login.scss

```diff
@@ -0,0 +1,7 @@
+.center {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+  width: 100vw;
+}
```

Comment:
Classname should be distinct enough, maybe start your classname with `login`?

Reply:
Changed appropriately